spec/rjgit_spec.rb

```diff
@@ -170,6 +170,11 @@
         end
       end
 
+    it "mimics git-describe" do
```

Comment:
Please also add tests for negative cases (wrong input, or no `no names found`, etc.).